procrastinate/sql/migrations/02.14.01_01_add_indexes_for_fetch_job.sql

```diff
@@ -0,0 +1,7 @@
+-- recreate procrastinate_jobs_id_lock_idx index by adding aborting status to the filter so that it can be used by the fetch job function
+CREATE INDEX procrastinate_jobs_id_lock_idx_temp ON procrastinate_jobs (id, lock) WHERE status = ANY (ARRAY['todo'::procrastinate_job_status, 'doing'::procrastinate_job_status, 'aborting'::procrastinate_job_status]);
```

Comment:
Note for self, don't forget to tell people in the release notes to create the index concurrently if applicable.